Dockerfile

```diff
@@ -18,15 +18,32 @@ COPY --chown=app . .
 RUN npm config set update-notifier false
 RUN npm ci --ignore-scripts && npm run compile
 
-RUN if [ "$SKIP_TESTS" = 'false' ]; then ln -s $APP_HOME/tests/fixtures env && npm test && rm env; fi
+RUN set -e && \
+    npm config set update-notifier false && \
+    npm ci --ignore-scripts && \
+    npm run compile
```

Comment:
Good idea to run them all in a single command, also to reduce unnecessary layers. But these are now duplicated with above lines 18-19.